GCEWindowsAgent/metadata.go

```diff
@@ -52,8 +52,10 @@ type projectJSON struct {
 
 type attributesJSON struct {
 	WindowsKeys           string `json:"windows-keys"`
+	DiagnosticsKeys       string `json:"diagnostics"`
 	DisableAddressManager string `json:"disable-address-manager"`
 	DisableAccountManager string `json:"disable-account-manager"`
+	DisableDiagnostics    string `json:"disable-diagnostics"`
```

Comment:
I think this should be `enable-diagnostics` instead of disable. We should be not included this behavior by default.

Reply:
Done.